pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -17,6 +17,7 @@
 
 # for determining the latest compatible linuxPackages
 , linuxPackages_5_19 ? pkgs.linuxKernel.packages.linux_5_19
+, linuxPackages_6_0 ? pkgs.linuxKernel.packages.linux_5_20
```

Comment:
does it make sense to rename this attribute to `latestCompatibleLinuxPackages`? and get rid of `linuxPackages_5_19`

Reply:
Maybe, but a cleanup of the derivation is beyond the scope of this PR.

Reply:
Good question. I think it would be an improvement, personally.